lib/phoenix/digester.ex

```diff
@@ -46,18 +47,48 @@ defmodule Phoenix.Digester do
     |> Enum.map(&(map_file(&1, input_path)))
   end
 
-  defp generate_manifest(files, output_path) do
+  defp load_digests(input_path) do
+    manifest_path = Path.join(input_path, "manifest.json")
```

Comment:
Why is that? existing digested files should be in the output path as well, right?